Networking/Networking/Remote/GenerativeContentRemote.swift

```diff
@@ -223,6 +223,20 @@ private extension GenerativeContentRemote {
                 "don’t suggest any value other than the available ones."
         }()
 
+        let shippingJson = {
+            guard let dimensionUnit,
+                  let weightUnit else {
+                return ""
+            }
```

Comment:
❓ Do we really need to require both dimension and weight units? Is there any case where either of them are available? E.g., when only the weight unit is available, we can still ask AI to guess the weight right?

Reply:
Good thinking. Started asking based on available unit in e2f1054b01ffa508dfa5e4ad487afbf5c0bfcbc9 and added tests in d34e69194a30115e6b4184cb5fb13984e9901343 and d34e69194a30115e6b4184cb5fb13984e9901343